server/src/main/java/io/kroki/server/service/HealthHandler.java

```diff
@@ -41,6 +47,10 @@ public Handler<RoutingContext> create() {
       for (ServiceVersion serviceVersion : serviceVersions) {
         versions.put(serviceVersion.getService(), serviceVersion.getVersion());
       }
+      if (blockedThreadChecker != null) {
+        data.put("blockedWorkerPercentage", blockedThreadChecker.blockedWorkerThreadPercentage());
+        data.put("blockedEventLoopPercentage", blockedThreadChecker.blockedEventLoopThreadPercentage());
+      }
```

Comment:
I wonder if we should export data using Prometheus format on `/metrics`.

Reply:
I don't use Prometheus on my server but I'm happy to help. Can you create another issue and assign to me? I'll work on it when I have the bandwidth

Reply:
Thanks, I've created an issue